wonderful_bing/wonderful_bing.py

```diff
@@ -92,7 +102,6 @@ def download_and_set(self):
         if path.exists(picture_path):
             print("You have downloaded the picture before.")
             print("Have a look at it --> {0}".format(picture_path))
-            sys.exit()
```

Comment:
IMHO, I use this script to make sure bing wallpaper is set as my wallpaper, the picture should be set as wallpaper even if it's downloaded.